internal/domain/tdexFeeder.go

```diff
@@ -0,0 +1,65 @@
+package domain
```

Comment:
for file names, the main convention is to use underscore for file names, instead of camel case